src/effects/VST3/internal/PlugFrame.h

```diff
@@ -0,0 +1,34 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  @file PlugFrame.h
+
+  @author Vitaly Sverchinsky
+
+  @brief Part of Audacity VST3 module
+
+**********************************************************************/
+
+#pragma once
+
+#include <wx/wx.h>
+#include <pluginterfaces/gui/iplugview.h>
+
+namespace internal
+{
+
+   //Dispatches window resize events to the wxWindow
```

Comment:
Start the comment with `//!` and Doxygen will extract it as the brief description of the class that follows.

You can write a long comment for detail after the brief comment if you enclose it in
```
/*!
...
*/
```